src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -8,6 +8,8 @@
     <UseWpf>true</UseWpf>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>$(DefineConstants);EDITOR_FEATURES</DefineConstants>
+    <!-- For whatever reason doesn't work in this project. -->
+    <RestoreUseStaticGraphEvaluation>false</RestoreUseStaticGraphEvaluation>
```

Comment:
If you added this because of CI failures in Correctness_Rebuild leg, that's probably an unrelated issue - happening in all Roslyn PRs now; this should fix it: https://github.com/dotnet/roslyn/pull/78437

Reply:
OMG 🗡️ I spent about an hour trying to understand the issue. For whatever reason one of the EditorFeatures evaluations in the binlog doesn't have expected items.

Thanks for letting me know. I will revert this change then.